nixos/modules/services/networking/pihole-ftl.md

```diff
@@ -0,0 +1,53 @@
+# pihole-FTL {#module-services-networking-pihole-ftl}
+
+*Upstream documentation*: <https://docs.pi-hole.net/ftldns/>
+
+pihole-FTL is a fork of [Dnsmasq](#module-services-networking-dnsmasq),
+providing some additional features, including an API for analysis and
+statistics.
+
+This module uses the configuration [options of the Dnsmasq
+module](#module-services-networking-dnsmasq).
```

Comment:
```suggestion
This module uses the configuration {#module-services-networking-dnsmasq}.
```
also we don't need to write so shotr lines, that a simple sentence no longer fits onto it.

Reply:
Fair, my editor settings are causing this. Resolved now, since I changed the link format.